src/PHPUnit/Controller/AbstractHttpControllerTestCase.php

```diff
@@ -413,30 +413,28 @@ public function registerXpathNamespaces(array $xpathNamespaces)
      *
      * @param  string $path
      * @param  bool $useXpath
-     * @return Document\NodeList
+     * @return Crawler
      */
     private function query($path, $useXpath = false)
     {
         $response = $this->getResponse();
-        $document = new Document($response->getContent());
+        $document = new Crawler($response->getContent());
 
         if ($useXpath) {
-            $document->registerXpathNamespaces($this->xpathNamespaces);
```

Comment:
Not sure what used for. Not covered test to accept my changes. Anyone?

Reply:
It basically sets the scope for XML search operations: without this, some elements would not necessarily be discoverable OOTB.

Reply:
I have added test case but I cannot make it pass. Is test or implementation wrong?

Reply:
Copied example from Symfony webpage works!